DataFormats/GEMRecHit/interface/GEMSegment.h

```diff
@@ -40,8 +40,8 @@ class GEMSegment final : public RecSegment {
              const LocalVector& direction,
              const AlgebraicSymMatrix& errors,
              double chi2,
```

Comment:
Now you pass everything as `float`: is it still necessary to pass chi2 as a `double`?

Reply:
@perrotta I think this was originally to match the interface of CSCSegment, which also defines theChi2 as double. Is it prefered to just switch to float here?

Reply:
If using a double is any useful in the code it can stay as such. If not, better to reduce precision (and also make it uniform with the other similar parameters)

Reply:
In GEMCSCSegment, we combine the chi2 from CSC and GEM, so I would have thought it would be better to keep it the same precision in this case. @jshlee do you have an opinion?